flow_judge/models/adapters/baseten/config.yaml

```diff
@@ -0,0 +1,41 @@
+model_name: "Flow-Judge-v0.1"
+# Note: prefix_caching is not supported by vLLM for this model, please do not include enable_prefix_caching as part of the vllm config in config.yaml
+environment_variables:
+  {
+    "CUDA_HOME": "/usr/local/cuda",
+    "PATH": "/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+    "LD_LIBRARY_PATH": "/usr/local/cuda/extras/CUPTI/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64",
+    "VLLM_ATTENTION_BACKEND": "FLASH_ATTN",
+  }
+external_package_dirs: []
+python_version: py311
+base_image:
+  image: baseten/truss-server-base:3.11-gpu-v0.9.0
+model_metadata:
+  example_model_input:
```

Comment:
This should be something we would actually use, ie from the notebooks or from Notion where B created examples to use.